shepherd/api/views.py

```diff
@@ -65,7 +68,13 @@ def is_ready(job_id: str):
         :param job_id: An identifier of the queried job
         """
         check_job_exists(minio, job_id)
-        return JobStatusResponse({'ready': shepherd.is_job_done(job_id)})
+        ready = shepherd.is_job_done(job_id)
+        if ready:
+            timestamp = datetime.fromtimestamp(calendar.timegm(minio.stat_object(job_id, 'done').last_modified))
```

Comment:
Could you break this down into ~2 statements? The nested calls are kinda hard to read.